cmd/gb/info.go

```diff
@@ -32,22 +32,27 @@ Values:
 		The value of runtime.GOROOT for the Go version that built this copy of gb.
 
 info returns 0 if the project is well formed, and non zero otherwise.
+If one or more variable names is given as arguments, info prints the 
+value of each named variable on its own line.
 `,
-		Run:      info,
+		Run: info,
+		ParseArgs: func(ctx *gb.Context, cwd string, args []string) []string {
+			return args
+		},
 		AddFlags: addBuildFlags,
 	})
 }
 
 func info(ctx *gb.Context, args []string) error {
-	vars := []struct{ name, value string }{
-		{"GB_PROJECT_DIR", ctx.Projectdir()},
-		{"GB_SRC_PATH", joinlist(ctx.Srcdirs()...)},
-		{"GB_PKG_DIR", ctx.Pkgdir()},
-		{"GB_BIN_SUFFIX", ctx.Suffix()},
-		{"GB_GOROOT", runtime.GOROOT()},
+	env := makeenv(ctx)
+	if len(args) > 0 {
+		for _, arg := range args {
+			fmt.Println(findenv(env, arg))
```

Comment:
I think there should be a check here that if findenv returns "", skip the output.